src/app/components/table/table.ts

```diff
@@ -2511,7 +2511,14 @@ export class Table implements OnInit, AfterViewInit, AfterContentInit, Blockable
 
     onColumnResizeBegin(event: any) {
         let containerLeft = DomHandler.getOffset(this.containerViewChild?.nativeElement).left;
-        this.resizeColumnElement = event.target.parentElement;
+        let resizeThElement = event.target.parentElement;
+        while (resizeThElement.tagName !== 'TH') {
```

Comment:
Why do you use `event.target.closest('th')`?

Reply:
You are absolutely right, I should have done that! :)
Fixed!